tests/server.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test } from "@playwright/test";
+import {
+  ServerSettingType,
+  activateSetting,
+  resetSetting,
+  updateSetting,
+  visitServerSettings,
+} from "./helpers/server";
+
+const settings: {
+  settingName: string;
+  settingType: ServerSettingType;
+  content?: string;
+}[] = [
+  { settingName: "acme.agree_tos", settingType: "checkbox" },
+  {
+    settingName: "acme.ca_url",
+    settingType: "text",
+    content: "https://acme-v02.api.letsencrypt.org/directory/test",
+  },
+  {
+    settingName: "acme.domain",
+    settingType: "text",
+    content: "test.com",
+  },
+  {
+    settingName: "acme.email",
+    settingType: "text",
+    content: "test@test.com",
+  },
+  {
+    settingName: "loki.auth.password",
+    settingType: "password",
+    content: "abc123",
+  },
+  {
+    settingName: "user.ui_title",
+    settingType: "text",
+    content: "Test User",
+  },
```

Comment:
Thinking about this again -- maybe enough to keep just 1 setting per type: checkbox, text and password?

Reply:
sure, we can do that :+1: